src/App.jsx

```diff
@@ -47,6 +47,8 @@ export function App() {
 		});
 	}, [listToken]);
 
+	console.log(data);
+
```

Comment:
We don't want this `console.log`, do we?

```suggestion
```

Reply:
Thanks for catching that!